docs/modules/deployment/pages/upgrade/basic.adoc

```diff
@@ -0,0 +1,134 @@
+
+[[upgrade-basic]]
+= Basic Upgrade
+
+This procedure describes how to do a basic {page-component-title} upgrade, including upgrading the PostgreSQL database to a new version.
+You may need to complete additional steps to upgrade in a more complex setup (running more than one OpenNMS instance, database migration requirements, age of current version, and so on).
+
+IMPORTANT: Make sure you complete the tasks in the xref:deployment:upgrade/introduction.adoc#byb_upgrade[before you begin] section before upgrade.
+
+== PostgreSQL upgrade and configuration
+This example illustrates an upgrade from PostgreSQL 9.4 to 12.
+The process should be similar for different versions.
+
+. Become root:
+`sudo su -`
++
+Expected outcome: `whoami` returns `root`.
+
+. Install the latest version of the PostgreSQL repository package:
+`yum -y install <repository package location>`
++
+Expected outcome: `yum repolist pdg*` reports available repos for postgresql versions 9.6 through 14.
+
+. Install new PostgreSQL version: `yum -y install postgresql-12 postgresql-12-server`
++
+Upstream versions are designed to coexist with multiple installed versions.
++
+Expected outcome: `rpm -qa | grep postgresql` shows both 9.4 and 12.x package versions installed.
+
+. Initialize the PostgreSQL database: `/usr/bin/postgresql-12-setup initdb`
++
+Expected result: command completes with no errors.
+
+. Stop OpenNMS: `systemctl stop opennms.service`
++
+Expected result: `systemctl status opennms.service` reports opennms is stopped.
++
+NOTE: Your OpenNMS downtime starts at this point.
+
+. Stop earlier version of PostgreSQL: `systemctl stop postgresql`
++
+Expected result: `systemctl status postgresql.service` reports postgresql is stopped.
+
+. Disable earlier version PostgreSQL: `systemctl disable postgresql`
+. Stop PostgreSQL 12: `systemctl postgresql-12 stop`
++
+Expected result: `systemctl status postgresql-12.service` reports postgresql-12 is stopped.
+
+. Become the `postgres` user: `sudo su – postgres`
++
+Expected result: `whoami` returns `postgres`
+
+. Run the PostgreSQL upgrade in "check" mode: `/var/lib/pgsql/data -D /var/lib/pgsql/12/data -r -v -c`
```

Comment:
The database migration steps were copied nearly verbatim from https://opennms.discourse.group/t/upgrading-postgresql/772#upgrading-on-rhel-or-centos-using-pg_upgrade-6 

There may be other useful information in that post for reference.